addons/mail/static/src/web/chat_window/chat_window.scss

```diff
@@ -9,8 +9,14 @@
         height: $o-mail-Discuss-headerHeight;
     }
 
-    .o-mail-ChatWindow-command:hover {
-        background-color: rgba(0, 0, 0, 0.075);
+    .o-mail-ChatWindow-command {
+        &:hover {
+            background-color: rgba(0, 0, 0, 0.075);
+        }
+
+        i {
+            width: 30px;
```

Comment:
If the goal is to have the icons take the same width, shouldn't we use fa-fw instead?